CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 3.5.1)
+cmake_minimum_required(VERSION 3.5.2)
```

Comment:
Not all developers use 3.10.2. 
It is **minimal required**, but actual version can be arbitrary (higher).

Reply:
It would be hard to find an error in case some feature from CMake 3.5+ is used, since there will be no error in container build. So either docker CMake version should be 3.5.2, in which case SWIG build should fail due to some missing functions, or version in CMakeLists.txt should be 3.10.2

Reply:
If you use any functionality that requires cmake of higher version --  you are welcome to change it. 
I use NuDB, which requires 3.5.2. If SWIG requires cmake 3.7+, someone should conduct tests and make according fix. It is not a part of this task.

Reply:
Please increase the version so that we won't have a compatibility problem with development dependency (SWIG) @Warchant

Reply:
swig should be unrelated to that pr, why that fixes should be here?